db-engine/src/main/java/com/splicemachine/db/impl/token/KeyAccessTokenVerifier.java

```diff
@@ -0,0 +1,80 @@
+package com.splicemachine.db.impl.token;
+
+import com.splicemachine.db.authentication.AccessTokenVerifier;
+import com.splicemachine.db.iapi.error.StandardException;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.security.Key;
+import java.security.KeyStore;
+
+public class KeyAccessTokenVerifier implements AccessTokenVerifier {
+
+    public static final String SPLICE_JWT_TOKEN_JKS_KEY_STORE = "splice.jwtToken.jksKeyStore";
+    public static final String SPLICE_JWT_TOKEN_JKS_KEYSTORE_PASSWORD = "splice.jwtToken.jksKeystore.password";
+    public static final String SPLICE_JWT_TOKEN_JKS_KEYSTORE_ALIAS = "splice.jwtToken.jksKeystore.alias";
+
+    private final String keyStorePath;
+    private final String keyStorePassword;
+    private final String keyStoreAlias;
+    private final String issuer;
+
+    /**
+     *                 splice.jwtToken.jksKeyStore
+     *                 splice.jwtToken.jksKeystore.password
+     *                 splice.jwtToken.jksKeystore.alias
+     */
+    protected KeyAccessTokenVerifier() {
+        keyStorePath = System.getProperty(SPLICE_JWT_TOKEN_JKS_KEY_STORE);
+        keyStorePassword = System.getProperty(SPLICE_JWT_TOKEN_JKS_KEYSTORE_PASSWORD);
+        keyStoreAlias = System.getProperty(SPLICE_JWT_TOKEN_JKS_KEYSTORE_ALIAS);
```

Comment:
I don't think we want these as System properties, but rather as configuration parameters

Reply:
changed to use a configuration. As I moved the code to EE/splice-auth, I needed to add some more dependencies to this module: hbase_storage, hadoop-common, hbase-client to use HConfiguration